spec/fx/adapters/mysql/functions_spec.rb

```diff
@@ -17,7 +17,7 @@
       expect(first.name).to eq "test"
       expect(first.definition).to eq <<-EOS.strip_heredoc.chomp
         DELIMITER $$
-        CREATE FUNCTION `test`() RETURNS text CHARSET utf8mb4
+        CREATE FUNCTION `test`() RETURNS text CHARSET #{ActiveRecord::Base.connection.charset}
```

Comment:
Nit: The `connection` variable above should be reused.

Reply:
Reused and added function with parameter